content/doc/book/managing/system-properties.adoc

```diff
@@ -606,9 +606,31 @@ The following placeholders are supported for this value:
 * *$\{ITEM_FULL_NAME}* – The full name of the item, with file system unsafe characters replaced by others.
 * *$\{ITEM_FULLNAME}* – See above, but does not replace unsafe characters. This is a legacy option and should not be used.
 
-+
```

Comment:
This preserved the indentation of the following paragraph.

Reply:
I was trying to fix this today, and got it partially fixed using this WIP: https://github.com/jenkins-infra/jenkins.io/compare/master...rkivisto:fix-pr-3387?expand=1

But I'm still having troubles with  the bullet points not lining up after my WIP:
![Jenkins_Features_Controlled_with_System_Properties](https://user-images.githubusercontent.com/19917557/83310205-3eb13f00-a1d9-11ea-9541-e33fa3992cfc.png)

Thank you for working on a more fundamental fix in https://github.com/jenkins-infra/jenkins.io/pull/3393

I'm going to review that PR and please let me know if I can help.